app/src/main/java/org/simple/clinic/home/overdue/search/OverdueSearchScreen.kt

```diff
@@ -176,7 +176,8 @@ class OverdueSearchScreen : BaseScreen<
       .build()
 
   override fun uiRenderer() = OverdueSearchUiRenderer(
-      ui = this
+      ui = this,
+      isOverdueSelectAndDownloadEnabled = features.isEnabled(Feature.OverdueSelectAndDownload) && country.isoCountryCode == Country.INDIA
```

Comment:
I think we would want to show download option for search results, even if the select and download is disabled. Can you check with design once to confirm? 

If that's the case, then we have to bump the instant search version instead.